hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableTokenRelationStore.java

```diff
@@ -69,33 +69,32 @@ public void commit() {
     }
 
     /**
-     * Returns the {@link TokenRelation} with the given number. If no such token relation exists, returns {@code Optional.empty()}
+     * Returns the {@link TokenRelation} with the given entity numbers. If no such token relation exists, returns {@code Optional.empty()}
      *
-     * @param tokenNum - the number of the token relation to be retrieved
      * @param accountNum - the number of the account relation to be retrieved
+     * @param tokenNum   - the number of the token relation to be retrieved
      */
-    public Optional<TokenRelation> get(final long tokenNum, final long accountNum) {
+    public Optional<TokenRelation> get(final long accountNum, final long tokenNum) {
```

Comment:
We should probably make this to avoid confusion and mistakes. Similary in `getForModify`
```
public Optional<TokenRelation> get(final AccountID accountId final TokenID tokenNum) {
```